aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/DeleteHandler.java

```diff
@@ -0,0 +1,54 @@
+package software.amazon.rds.customdbengineversion;
+
+import software.amazon.awssdk.services.rds.RdsClient;
+import software.amazon.awssdk.services.rds.model.CustomDbEngineVersionNotFoundException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.rds.common.handler.Commons;
+import software.amazon.rds.common.handler.HandlerConfig;
+
+public class DeleteHandler extends BaseHandlerStd {
+
+
+    public DeleteHandler() {
+        this(CUSTOM_ENGINE_VERSION_HANDLER_CONFIG_10H);
+    }
+
+    public DeleteHandler(HandlerConfig config) {
+        super(config);
+    }
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+            final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request,
+            final CallbackContext callbackContext,
+            final ProxyClient<RdsClient> proxyClient,
+            final Logger logger) {
+        return proxy.initiate("rds::delete-custom-db-engine-version", proxyClient, request.getDesiredResourceState(), callbackContext)
+                .translateToServiceRequest(Translator::deleteCustomDbEngineVersion)
+                .backoffDelay(config.getBackoff())
+                .makeServiceCall((deleteDbClusterEndpointRequest, proxyInvocation) ->
```

Comment:
Why would we call it an endpoint request? Was it meant to be engine version request?

Reply:
This is a variable name only, looks like an oversight :)